python/sbp/client/loggers/json_logger.py

```diff
@@ -62,13 +67,17 @@ def next(self):
 
     """
     for line in self.handle:
-      data = json.loads(line)
-      delta = data['delta']
-      timestamp = data['timestamp']
-      item = SBP.from_json_dict(data['data'])
       try:
-        yield (delta, timestamp, self.dispatcher(item))
-      except KeyError:
-        yield (delta, timestamp, item)
+        data = json.loads(line)
+        delta = data['delta']
+        timestamp = data['timestamp']
+        item = SBP.from_json_dict(data['data'])
+        try:
+          yield (delta, timestamp, self.dispatcher(item))
+        except KeyError:
+          yield (delta, timestamp, item)
+      except ValueError:
```

Comment:
Do we want to have different handling based on the kind of `ValueError` we have? Two cases it seems: 

```python
# 1. scan error implies end of file, so exit. 
ValueError: Unterminated string starting at:                                
# 2. value error implies bad entry, but the file may not be over yet? 
ValueError: Out of range float values are not JSON compliant   
```  

For 2, does the error file originating the issue continue past that point, or is it another manifestation of an unwanted EOF? There's a case to be made for exiting early, because it implies something serious about the data coming out. I suspect we'd want to be a bit more robust here. If we passed back an SBP object using just the SBP fields, a downstream analysis test could count these issues?